src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -1,7 +1,13 @@
 package uk.gov.companieshouse.company.profile.service;
 
+import com.mongodb.client.result.UpdateResult;
 import java.util.NoSuchElementException;
 import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
Unused import, but for some reason checkstyle is not flagging it for me either - IntelliJ is